references/firestore-facade/sf-firestore-facade-lookup-v1/sharedflowbundle/resources/jsc/JS-SetFirebaseMockedCacheVariables.js

```diff
@@ -0,0 +1,28 @@
+ /**
+  Copyright 2023 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+// set variable *** flow.lookup.iscontent.cached ***
+context.setVariable('flow.lookup.iscontent.cached','true');
```

Comment:
See comments for [the other JS file](https://github.com/apigee/devrel/pull/701/files#diff-a6f1a53477a0df107fc7aebe164e76ae06dc6c633260bfccbeb8a193de1b3b24)

Reply:
done